src/js/resources.js

```diff
@@ -35,6 +37,7 @@ module.exports = {
     },
 
     urls: urls,
+    root: root,
```

Comment:
When I read [this line](https://github.com/PEDSnet/pedsnet-catalogue/pull/12/files#diff-c9bf81348e383fe276f7b145b58c76d5R29), putting this in the `resources` module feels misleading. This is not the *root* of the resources, but of the client itself. I think adding a `env` module that has the sole job of copying `window.catalogue` and exporting it would be appropriate. All other modules that depend on the environment variables or configuration would import that module to access the values.